modules/ossm-federation-config-smcp.adoc

```diff
@@ -0,0 +1,397 @@
+
+.Federation DNS parameters
+[source,yaml]
+----
+kind: ServiceMeshControlPlane
+metadata:
+  name: red-mesh
+  namespace: red-mesh-system
+spec:
+  techPreview:
+    meshConfig:
+      defaultConfig:
+        proxyMetadata:
+          ISTIO_META_DNS_CAPTURE: "true"
+          PROXY_XDS_VIA_AGENT: "true"
+----
+
+.Federation DNS parameters
+[options="header"]
+[cols="l, a, a, a"]
+|===
+|Parameter |Description |Values |Default value
+|ISTIO_META_DNS_CAPTURE:
+|This setting is required so that the proxy handles DNS requests, which are delegated to Istiod.
+|`true`/`false`
+|`true`
+
+|PROXY_XDS_VIA_AGENT:
+|This setting is required to enable DNS capture.
+|`true`/`false`
+|`true`
+|===
+
+
+////
+TODO
+.Sample SMCP green mesh
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion:
+kind:
+metadata:
+spec:
+----
+====
+
+
+.Sample SMCP blue mesh
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion:
+kind:
+metadata:
+spec:
+----
+====
+////
+
+.Procedure from the Console
+
+Follow this procedure to edit the `ServiceMeshControlPlane` with the {product-title} web console. This example uses the `red-mesh` as an example.
+
+. Log in to the {product-title} web console as a user with the cluster-admin role.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. Click the *Project* menu and select the project where you installed the control plane. For example, `red-mesh-system`.
+
+. Click the {ProductName} Operator.
+
+. On the *Istio Service Mesh Control Plane* tab, click the name of your `ServiceMeshControlPlane`, for example `red-mesh-install`.
+
+. On the *Create ServiceMeshControlPlane Details* page, click `YAML` to modify your configuration.
+
+. Modify your `ServiceMeshControlPlane` to add federation ingress and egress gateways and to specify the trust domain.
+
+. Click *Save*.
+
+
+.Procedure from the CLI
+
+Follow this procedure to create or edit the `ServiceMeshControlPlane` with the command line.  This example uses the `red-mesh` as an example.
+
+. Log in to the {product-title} CLI as a user with the `cluster-admin` role. Enter the following command. Then, enter your username and password when prompted.
++
+[source,terminal]
+----
+$ oc login --username=NAMEOFUSER https://{HOSTNAME}:6443
+----
++
+. Change to the project where you installed the control plane, for example red-mesh-system.
++
+[source,terminal]
+----
+$ oc project red-mesh-system
+----
++
+. Edit the `ServiceMeshControlPlane` file to add federation ingress and egress gateways and to specify the trust domain.
+
+. Run the following command to edit the control plane where `red-mesh-system` is the system namespace and `red-mesh-install.yaml` includes a full path to the file you edited:
++
+[source,terminal]
+----
+$ oc edit -n red-mesh-system -f red-mesh-install.yaml
+----
++
+. Execute the following command,where `red-mesh-system` is the system namespace, to see the status of the control plane installation.
```

Comment:
```suggestion
. Enter the following command, where `red-mesh-system` is the system namespace, to see the status of the control plane installation.
```
IBM SG "execute: Use only if a simpler term, such as "run", "issue", "start", or "enter" is inappropriate."